libr/io/p/io_ar.c

```diff
@@ -94,6 +94,7 @@ RIOPlugin r_io_plugin_ar = {
 	.name = "ar",
 	.desc = "Open ar/lib files [ar|lib]://[file//path]",
 	.license = "LGPL3",
+	.uri_handlers = "ar://, lib://",
```

Comment:
Why don't you use an actual list rather than a plain `char *`?

Reply:
const char ** should be fine, but we can also use comma separated values and use r_str_split_list, but then i would not add a space after the comma